actk/steps/single_cell_images/single_cell_images.py

```diff
@@ -333,6 +337,12 @@ def run(
             # Block until all complete
             bbox_results = handler.gather(bounding_box_futures)
 
+            if bbox:
+                num_of_channels = bbox_results[0][0]
+                bbox = [num_of_channels] + list(bbox)
+                bbox_results = [bbox for i in range(len(bbox_results))]
+                log.info(f"Using hard coded bounding box with ZYX dimensions: {bbox}.")
+
```

Comment:
Is this the only place the `bbox` parameter get used?  I'm unclear on how the cropping happens -- to me 
```bbox_results = [bbox for i in range(len(bbox_results))]```
looks like a `len(bbox_results)` list of just the same tuple, e.g. `[(64, 168, 104), (64, 168, 104), ... ]`

Reply:
I think the cropping is happening via aicsimageprocessing.cell_rigid_registration - 

`https://github.com/AllenCellModeling/actk/blob/1d4ae5fc01b62b9bb35789aae5efa72e46d4b6a5/actk/steps/single_cell_images/single_cell_images.py#L142-L147`, 

and the bounding box is computed here (averaged across all registered image sizes) -
https://github.com/AllenCellModeling/actk/blob/1d4ae5fc01b62b9bb35789aae5efa72e46d4b6a5/actk/steps/single_cell_images/single_cell_images.py#L339-L342

before being passed into the distributed handler via _generate_single_cell_images. So I thought the best place to add the bbox argument would be here. I am broadcasting bbox across length len(bbox_results) to be consistent with the bbox_result shape collected without the hard coded box parameters.

Let me know if this doesn't make sense and I should change this!